### DISCUSSION TITLE
Game server status card for RconConnect

### DISCUSSION THREAD

**Discussion by pfistdo** [gameserverapp/Platform#638] [Suggestions]
Implement the game server status card for users with RconConnect. The game server status log would also be a nice feature to have for RconConnect. I don't know if this is possible to implement but I would love to be able to use this feature.

**Comment by pfistdo** [comment]
Also maybe consider displaying the player count or even the player names in a Discord widget? Maybe it would also be possible to display this information in-game with one of the F keys still available.

**Comment by mvaessen** [comment]
Status cards are not supported for RconConnect unfortunately.
